CHANGELOG.md

```diff
@@ -70,6 +70,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **CUMULUS-1912**
   - Updated reconciliationReports endpoints to query against Elasticsearch, delete report from both database and s3
   - Added `@cumulus/api-client/reconciliationReports`
+- **CUMULUS-1952**
+  - Updated HTTP(S) provider to accept username/password for Basic authorization. This change adds support for Basic Authorization such as Earthdata login redirects to ingest (i.e. as implemented in SyncGranule), but not to discovery (i.e. as implemented in DiscoverGranules). Discovery still expects the provider's file system to be publicly accessible, but not the individual files and their contents.
```

Comment:
I suppose we didn't implement this for discovery given that it was outside the scope of the request?

Reply:
Yes, outside of the scope. Also a completely separate implementation because we use a completely different library. But the ticket and user both indicate no need for discovery support.